low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -168,6 +168,7 @@ SERVICEMONITOR:
       - scsi-network-relay.service
       - kafka.service
       - kafka-zookeeper.service
+      - cortx_message_bus.service
```

Comment:
THis is CORTX service, which will be monitored by HA (pcs). Why the exception for this service, to be monitored by SSPL ? @rahul-tripathi-git 
cc @ajaysrivas

Reply:
As per Ajay's reply on jira, this should happen through pcs. @rahul-tripathi-git , sync with HA team. SSPL will wait till monday EOD to conclude on this. May not merge in given sprint